.circleci/config.yml

```diff
@@ -1,5 +1,32 @@
 version: 2.0
 jobs:
+    lint-js-css:
```

Comment:
Can we break this up into separate `lint-js`, `lint-css`, and `lint-audit` jobs like Travis had? That's make it a bit easier to see exactly which job causes a PR to fail.

Reply:
Updated to break them up.  Took the opportunity to also refactor for latest CircleCI version, with Orbs for less boilerplate and faster builds.